Vagrantfile

```diff
@@ -8,9 +8,11 @@ VAGRANTFILE_API_VERSION = "2"
 $ubuntu_setup_script = <<SETUP_SCRIPT
 cd /vagrant
 sudo ./bootstrap/ubuntu.sh
+
+# For some reason running setup.py w/o virtualenv in vagrant fails
 if [ ! -d "venv" ]; then
   virtualenv --no-site-packages -p python2 venv
-  ./venv/bin/pip install -r requirements.txt -e .[dev,docs,testing]
+  ./venv/bin/python setup.py install
```

Comment:
Urm...  `requirements.txt` is expected to be read. Also, since Vangrant is meant to be a dev environment, you want to install the `dev`, `docs` and `testing` setuptools extras. They are not installed when `python setup.py install` is run.